client/landing/stepper/declarative-flow/internals/steps-repository/import/index.tsx

```diff
@@ -12,8 +12,14 @@ import { BASE_ROUTE } from './config';
 import { generateStepPath } from './helper';
 import type { Step } from '../../types';
 import './style.scss';
+import type { ImporterPlatform } from 'calypso/lib/importer/types';
```

Comment:
Nit: should this be before the style import
```suggestion
import type { ImporterPlatform } from 'calypso/lib/importer/types';
import './style.scss';
```